edb/eschema/parser/grammar/declarations.py

```diff
@@ -132,6 +132,16 @@ def reduce_RowRawString(self, *kids):
         self.val = kids[0].parse_as_typelist()
 
 
+class TypeExpr(Nonterm):
```

Comment:
Is the shadowing of the previous definition of the `TypeExpr` production intentional?

Reply:
This is the only `TypeExpr` in eschema grammar.

Reply:
Nevermind, this is a different grammar.